Smoke test testJFRAvailable failed with jdk11 aix hotspot

Smoke test testJFRAvailable failed with jdk11 aix hotspot

```
03:34:30  FAILED: testJFRAvailable
03:34:30  java.lang.AssertionError: Expected JFR to be present but it is absent. expected [0] but found [1]
03:34:30  	at org.testng.Assert.fail(Assert.java:96)
03:34:30  	at org.testng.Assert.failNotEquals(Assert.java:776)
```
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-aix-ppc64-hotspot_SmokeTests/5/console

JDK Flight recorder was added to JDK 11 (JEP 328) and backported to JDK 8u262.
https://openjdk.java.net/jeps/328
https://bugs.openjdk.java.net/browse/JDK-8223147

--- a/test/functional/buildAndPackage/src/net/adoptopenjdk/test/FeatureTests.java
+++ b/test/functional/buildAndPackage/src/net/adoptopenjdk/test/FeatureTests.java
@@ -160,7 +160,9 @@ public class FeatureTests {
         }
         boolean shouldBePresent = false;
         if (jdkVersion.isNewerOrEqual(11) || jdkVersion.isNewerOrEqualSameFeature(8, 0, 262)) {
-            shouldBePresent = true;
+            if (!jdkPlatform.runsOn(OperatingSystem.AIX)) {
+                shouldBePresent = true;
+            }
         }
         LOGGER.info(String.format("Detected %s on %s, expect JFR to be present: %s",
                 jdkVersion, jdkPlatform, shouldBePresent));

